platform/linux-generic/pktio/socket_mmap.c

```diff
@@ -710,18 +713,22 @@ static int sock_mmap_init_global(void)
 	return 0;
 }
 
-const pktio_if_ops_t sock_mmap_pktio_ops = {
-	.name = "socket_mmap",
-	.print = NULL,
-	.init_global = sock_mmap_init_global,
-	.init_local = NULL,
-	.term = NULL,
+static pktio_ops_module_t socket_mmap_pktio_ops = {
+	.base = {
+		.name = "socket mmap",
```

Comment:
Again you've changed the interface name here from `socket_mmap` to `socket mmap`.  Keep the names unchanged as part of this PR to avoid application impact.

Reply:
Will fix all these.